.github/workflows/build-ci.yml

```diff
@@ -35,5 +35,5 @@ jobs:
       - name: Build image
         uses: docker/build-push-action@v1
         with:
-          repository: apclary/dungeon-revealer
+          repository: ${{ secrets.DOCKER_USERNAME }}/dungeon-revealer
```

Comment:
We should probably also update this to `dungeonrevealer`, just to be consistent.

Reply:
Ah, that one slipped through the cracks.